source/SIL.AppBuilder.Portal.Frontend/src/data/containers/with-current-user/hoc.tsx

```diff
@@ -71,7 +54,8 @@ export function withCurrentUser(opts = {}) {
 
         this.state = {
           currentUser: undefined, isLoading: true, error: undefined,
-          networkFetchComplete: false
+          networkFetchComplete: false,
+          fetchCurrentUser: this.fetchCurrentUser
```

Comment:
initially i was confused why fetchCurrentUser was being put into state, but I see that we are rendering this.state into currentUserProps which is then consumable by the Provider below.  got it.

Reply:
Yeah, I needed to expose this as a result of the provider wrapping the login component.
Auth0Lock async'ly sets the auth token, but upon redirect back from the auth0 service, it was trying to get the current user before any token at been set (cause that logic wraps auth0Lock), so, now, I just let that first request happen, and then as soon as the token is set, manually re-fetch the current user.